website/pages/en/index.js

```diff
@@ -0,0 +1,73 @@
+const React = require("react");
+
+function HomeSplash(props) {
```

Comment:
Might be good to put something here like logo at least and some general info
See features in readme

Reply:
No design needed but some info will be cool